### PR TITLE
first draft of Aint No Bugs on Me

### PR DESCRIPTION
**Pull request by r1b**

### CODE REVIEW

File: source/_posts/aint-no-bugs-on-me.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: Ain't No Bugs on Me
```
Comment (ptsteadman) [inline source/_posts/aint-no-bugs-on-me.md]: ...

File: source/_posts/aint-no-bugs-on-me.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+> There ain't no bugs on me
+> There may be bugs on some of you mugs
+> But there ain't no bugs on me
+
+*Traditional folk song*
+
+### Background
+
+How do you become aware of security bugs in your software systems? Is it when
+a machine prompts you to install updates? Is it when an engineer bumps the
```
Comment (ptsteadman) [inline source/_posts/aint-no-bugs-on-me.md]: Maybe you could add "Is it when an engineer... you could say 'when you read about it on Hacker News'...

File: source/_posts/aint-no-bugs-on-me.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+Creating an automated patch management system is a great way to stay secure.
+However, depending on your threat model, you might need to do even better than
+that. It may not be acceptable to wait for your patch management system to apply
+an update on a schedule or to wait for a patch in a library to propogate upstream.
+
+By keeping track of security vulnerabilities on your own, you can catch potential
+vulnerabilities as soon as they are made public. Software vendors and special
+interest groups have created a number of security feeds that makes it easy to stay
+abreast of the latest security bugs. In this article, we present a directory of
+the feeds that I follow at Computer Lab when managing critical systems.
```
Comment (ptsteadman) [inline source/_posts/aint-no-bugs-on-me.md]: Add TL;DR.

File: source/_posts/aint-no-bugs-on-me.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+are sent here.
+
+#### [Microsoft Security Updates](https://technet.microsoft.com/en-us/security/bulletins.aspx)
+
+On [Patch Tuesday](https://en.wikipedia.org/wiki/Patch_Tuesday), Microsoft releases
+information about patched vulnerabilities on its website. The bulletins are quite
+detailed, indicating the affected versions of any relevant software, mitigating
+factors, workaround & occasionally attribution to the original reporter.
+
+Message Frequency: Monthly
```
Comment (r1b) [inline source/_posts/aint-no-bugs-on-me.md]: Move to top

File: source/_posts/aint-no-bugs-on-me.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+### Bonus: Amazon Web Services
+
+I imagine that most readers are using AWS for some, if not all, of their infrastructure.
+Here are some resources to help you stay on top of vulnerabilities in the cloud.
+
+#### [Amazon Linux AMI Security](https://alas.aws.amazon.com/)
+
+Notifications of vulnerabilities in Amazon's flavor of Red Hat (Amazon Linux).
+
+Message Frequency: Varies
```
Comment (r1b) [inline source/_posts/aint-no-bugs-on-me.md]: same

File: source/_posts/aint-no-bugs-on-me.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+Format: RSS, Web
+
+#### [Amazon Security Bulletins](https://aws.amazon.com/security/security-bulletins/)
+
+Security bulletins for vulnerabilities in Amazon services (e.g: RDS) and
+internal software (e.g: Xen). Also includes general security notices for critical
+vulnerabilities a la CERT.
+
+Message Frequency: Varies
```
Comment (r1b) [inline source/_posts/aint-no-bugs-on-me.md]: same

**Pull request merged**